docs/kcl-std/types/std-types-Plane.md

```diff
@@ -1,13 +1,14 @@
 ---
 title: "Plane"
 subtitle: "Type in std::types"
-excerpt: "A plane."
+excerpt: "An abstract plane."
 layout: manual
 ---
 
-A plane.
-
+An abstract plane.
```

Comment:
what's the difference between abstract and concrete planes?

Reply:
Yeah, I wasn't sure what to call this, but I meant that it is not an object in the scene, it is just construction geometry, but I wasn't sure how to write that as an adjective